### PR TITLE
Prometheus: Metrics explorer add docs and image

### PR DESCRIPTION
**Pull request by bohandley**
**What is this?**
This is a pr to add documentation for the metrics explorer in the query builder docs.

**Why do we need this feature?**
The metrics explorer is being turned on by default in v10.1 and part of the roll out is to add documentation about the feature.

**Who is this feature for?**
The metrics explorer is for prometheus users with beginning and intermediate experience who want to explore their metrics.

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by gtk-grafana — APPROVED** [review]

File: docs/sources/datasources/prometheus/query-editor/index.md [hunk 1]
```diff
@@ -184,7 +204,7 @@ The autocompletion dropdown includes documentation for the suggested items where
 ### Metrics browser
 
 The metrics browser locates metrics and selects relevant labels to help you build basic queries.
-When you click **Metrics browser**, it displays all available metrics and labels.
+When you click **Metrics browser** in code mode, it displays all available metrics and labels.
```
Comment (lwandz13) [inline docs/sources/datasources/prometheus/query-editor/index.md:207]:
```suggestion
When you click **Metrics browser** in `Code` mode, it displays all available metrics and labels.
```

**Review by lwandz13 — CHANGES_REQUESTED** [review]
just a couple suggestions :-)

File: docs/sources/datasources/prometheus/query-editor/index.md [hunk 2]
```diff
@@ -126,6 +126,24 @@ When you are ready to create a query, you can choose the specific metric name fr
 The data source requests the list of available metrics from the Prometheus server based on the selected time rage.
 You can also enter text into the selector when the dropdown is open to search and filter the list.
 
+#### Metrics Explorer
+
+{{< figure src="/static/img/docs/prometheus/screenshot-grafana-prometheus-metrics-explorer-2.png" max-width="500px" class="docs-image--no-shadow" caption="Metrics explorer" >}}
+
+If you would like to explore your metrics further, you can open the **Metrics Explorer** by clicking the first option in the metric select.
+
+{{< figure src="/static/img/docs/prometheus/metrics-explorer-option.png" max-width="500px" class="docs-image--no-shadow" caption="Metrics explorer" >}}
```
[the remark this answers is not in the document]
Comment (bohandley) [reply docs/sources/datasources/prometheus/query-editor/index.md:135]: No, I think this was from a first version. This is a step in how to get to the metrics explorer through the metric select but according to the writer's toolkit we are not supposed to show a secondary navigation item (the option in the select to open the modal).

[the remark this answers is not in the document]
Comment (bohandley) [reply docs/sources/datasources/prometheus/query-editor/index.md]: Removed!

**Review by lwandz13 — APPROVED** [review]

**Pull request merged**